src/main/java/jmgomez/apipolicy/service/PolicyService.java

```diff
@@ -12,47 +12,29 @@
 @Service
 public class PolicyService implements IPolicyService {
 
-    private final PolicyClient policyClient;
-
     @Autowired
-    private ObjectMapper defaultObject;
+    private PolicyClient policyClient;
 
     @Autowired
-    public PolicyService(PolicyClient policyClient) {
-        this.policyClient = policyClient;
-    }
+    private MapStructService mapStructService;
 
     @Override
-    public List<Policy> getPolicies(String id) {
-        return policyClient.getPolicies(id);
+    public List<PolicyDto> getPolicies(String userId) {
+        return policyClient.getPolicies(userId).stream().map(policy -> mapStructService.mapPolicyToPolicyDto(policy)).toList();
     }
-
+    //@PreAuthorize("#id == authentication.name")
```

Comment:
Commented out code hurts my feelings. If you don't need it, remove it. Even if it was something pretty complicated you fear not to remember later... it's in git history.